web3.js/test/transaction.test.ts

```diff
@@ -3,14 +3,19 @@ import {Buffer} from 'buffer';
 import nacl from 'tweetnacl';
 import {expect} from 'chai';
 
+import { Connection } from '../src/connection';
 import {Keypair} from '../src/keypair';
 import {PublicKey} from '../src/publickey';
 import {Transaction, TransactionInstruction} from '../src/transaction';
 import {StakeProgram} from '../src/stake-program';
-import {SystemProgram} from '../src/system-program';
+import {SystemProgram, SYSTEM_INSTRUCTION_LAYOUTS} from '../src/system-program';
 import {Message} from '../src/message';
 import invariant from '../src/util/assert';
 import {toBuffer} from '../src/util/to-buffer';
+import { helpers, mockRpcResponse, mockServer } from './mocks/rpc-http';
+import { MOCK_PORT, url } from './url';
+import { LAMPORTS_PER_SOL } from '../src';
+import { stubRpcWebSocket } from './mocks/rpc-websockets';
```

Comment:
These imports can be cleaned up I think, looks like a few are unused?

Reply:
Yes, I thought I got it cleaned up but I missed some. I'll take care of it.